chain/chain/src/resharding.rs

```diff
@@ -263,6 +274,11 @@ impl Chain {
         let store = tries.get_store();
 
         let shard_id = shard_uid.shard_id();
+
+        RESHARDING_STATUS
+            .with_label_values(&[&shard_uid.shard_id.to_string()])
+            .set(ReshardingStatus::Finished.into());
```

Comment:
This should be `ReshardingStatus::BuildingState`

Reply:
We can add this to `build_state_for_split_shards` function as well if you'd like?

Reply:
For the labels why are we using shard_id instead of shard_uid?

Reply:
> This should be ReshardingStatus::BuildingState

thanks copy pasta my bad

> We can add this to build_state_for_split_shards function as well if you'd like?

I don't think we'll ever need it and we can always add it quickly. 

> For the labels why are we using shard_id instead of shard_uid?

It has more info in it and I feel better knowing for sure what shard layout version we're talking about.

Reply:
> It has more info in it and I feel better knowing for sure what shard layout version we're talking about.

Yes, exactly, we should be using shard_uid, the code currently has shard_id.

Reply:
Ah thanks, total brain fart on my side, I was totally intending to use shard uids but somehow ended up with shard id. Will fix soon.